src/js/angular/rest/ttyg.rest.service.js

```diff
@@ -52,17 +52,24 @@ function TTYGRestService($http) {
 
     /**
      * Exports a conversation by its ID.
-     *
-     * TODO: this should give the user a file to download, but the backend doesn't support it yet.
-     *
      * @param {string} id
-     * @return {*}
+     * @return {Promise<{Blob, string}>} Returns the conversation as a Blob and the filename wrapped in a Promise
      */
     const exportConversation = (id) => {
         if (DEVELOPMENT) {
             return _fakeBackend.exportConversation(id);
         }
-        return $http.post(`${CONVERSATIONS_ENDPOINT}/export/${id}`);
+        return $http.get(`${CONVERSATIONS_ENDPOINT}/export/${id}`, {responseType: 'blob'}).then(function (res) {
```

Comment:
Could you move this response transformation into ttyg.service.js? We usually perform response conversions to models in that file.

Reply:
I completely missed that there is a service in the middle :(
Moved it to the service.